code/renderers/vue3/template/stories/preview.js

```diff
@@ -0,0 +1,13 @@
+import globalThis from 'global';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { setup } from '@storybook/vue3';
+
+// TODO: I'd like to be able to export rather than imperatively calling an imported function
+// export const setup = (app) => {
+//   app.component('GlobalButton', Button);
+// };
```

Comment:
Is this a concept we want to add everywhere? Like a setup function in `preview.js`? Is there a use case where you wouldn't just immediately call it right away (in which case maybe having it would be adding complexity for its own sake)?

Reply:
cc @ndelangen